content/applications/general/auth/azure.rst

```diff
@@ -2,9 +2,159 @@
+
+This is particularly helpful if the organization uses Azure Workspace, and wants employees within
+the organization to connect to Odoo using their Microsoft Accounts.
+
+.. warning::
+   Databases hosted on Odoo.com should not use OAuth login for the owner or administrator of the
+   database as it would unlink the database from their Odoo.com account. If OAuth is set up for that
+   user, the database will no longer be able to be duplicated, renamed, or otherwise managed from
+   the Odoo.com portal.
+
 
 .. seealso::
    - :doc:`../../productivity/calendar/outlook`
    - :doc:`/administration/maintain/azure_oauth`
+
+Configuration
+=============
+
+Integrating the Microsoft sign-in function requires configuration on Microsoft and Odoo.
+
+Odoo System Parameter
+---------------------
+
+First activate the :ref:`developer mode <developer-mode>`, and then go to :menuselection:`Settings
+--> Technical --> System Parameters`.
+
+Click :guilabel:`Create` and on the new/blank form that appears, add the following system parameter
+`auth_oauth.authorization_header` to the :guilabel:`Key` field, and set the :guilabel:`Value` to
+`1`. Then click :guilabel:`Save` to finish.
+
+Microsoft Azure dashboard
+-------------------------
+
+Create a new application
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To get started, go to `Microsoft's Azure Portal <https://portal.azure.com/>`_. Log in with the
+:guilabel:`Microsoft Outlook Office 365` account if there is one, otherwise, log in with the
+personal :guilabel:`Microsoft account`. A user with administrative access to the Azure Settings must
+connect and perform the following configuration. Next, navigate to the section labeled
+:guilabel:`Manage Azure Active Directory`.
+
+Now, click on :guilabel:`Add (+)`, located in the top menu, and then select :guilabel:`App
+registration`. On the :guilabel:`Register an application` screen, rename the :guilabel:`Name` to
+`Odoo` or something recognizable. Under the :guilabel:`Supported account types` select
+:guilabel:`Accounts in this organizational directory only (Default Directory only - Single tenant)`.
+
+Under the :guilabel:`Redirect URL` section, select :guilabel:`Web` as the platform, and then input
+`https://<odoo base url>/auth_oauth/signin` in the :guilabel:`URL` field. The Odoo base :abbr:`URL
+(Uniform Resource Locator)` is the canonical domain at which your Odoo instance can be reached (e.g.
+mydatabase.odoo.com if you are hosted on Odoo.com) in the :guilabel:`URL` field. Then, click
+:guilabel:`Register`, and the application is created.
+
+Authentication
+~~~~~~~~~~~~~~
+
+Edit the new app's authentication by clicking on :guilabel:`Authentication` after being redirected
+to the application's settings from the previous step.
+
+Select the *tokens* that should be issued by the authorization endpoint by scrolling down the screen
+and check the boxes labeled: :guilabel:`Access tokens (used for implicit flows)` and :guilabel:`ID
+tokens (used for implicit and hybrid flows)`.
```

Comment:
what are these? why are they important? do they have a cost to use?
if so we should include an admonition block saying so.